import-scripts/import-genie-data-transfer-deployment-color.sh

```diff
@@ -0,0 +1,209 @@
+        if all_replicas_ready $DEPLOYMENT_COLOR ; then
+            return 0
+        fi
+        check_count=$(($check_count+1))
+    done
+    echo "Error : scaling for '$DEPLOYMENT_COLOR' deployments failed to reach the ready state after $REPLICA_READY_CHECK_MAX_CHECKCOUNT iterations with a period of $REPLICA_READY_CHECK_PAUSE_SECONDS seconds. Exiting"
+    exit 1
+}
+
+function switchover_ingress_rules_to_updated_database_deployment() {
+    # recently we have checked that the k8s config repo is up to date and matches the production specs
+    # TODO switch over ingress rules to the new deployment
+    # - alter selected services in ingress rules for genie.cbioportal.org and genie-private.cbioportal.org in cbio-ingress.yaml
+    echo "switching traffic over to the updated database deployment"
+}
+
+function check_in_changes_to_kubernetes_into_github() {
+    #TODO: implement commit and push .. report problems as error
+    echo "checking in configuration changes to github"
+}
+
+function main() {
+    validate_arguments $@
+    MANAGE_DATABASE_TOOL_PROPERTIES_FILEPATH=$1
+    DESTINATION_COLOR=$2
+    if [ "$DESTINATION_COLOR" == "blue" ] ; then
+        SOURCE_COLOR="green"
+    else
+        SOURCE_COLOR="blue"
+    fi
+    source /data/portal-cron/scripts/automation-environment.sh
+    source /data/portal-cron/scripts/clear-persistence-cache-shell-functions.sh
+    echo "starting import-genie-data-transfer-deployment-color.sh"
+    check_current_color_is $MANAGE_DATABASE_TOOL_PROPERTIES_FILEPATH $SOURCE_COLOR
+    /data/portal-cron/scripts/authenticate_service_account.sh public
+    declare -x PUBLIC_CLUSTER_KUBECONFIG="/data/portal-cron/pipelines-credentials/public-cluster-config"
+    check_that_cluster_config_git_repo_is_current
+    FULL_MINUS_1=$((FULL_REPLICA_COUNT-1))
+    echo "starting up initial minimal deployment of $DESTINATION_COLOR"
+    scale_deployment_to_N_replicas $SOURCE_COLOR $FULL_MINUS_1
+    scale_deployment_to_N_replicas $DESTINATION_COLOR 1
+    echo "increasing deployment of $DESTINATION_COLOR"
+    scale_deployment_to_N_replicas $SOURCE_COLOR 1
+    scale_deployment_to_N_replicas $DESTINATION_COLOR $FULL_MINUS_1
+    echo "synchronizing user tables"
+    /data/portal-cron/scripts/synchronize_user_tables_between_databases.sh /data/portal-cron/pipelines-credentials/manage_genie_database_update_tools.properties $SOURCE_COLOR $DESTINATION_COLOR
+    # TODO program a smart wait for cache clearing to complete
+    clearPersistenceCachesForGeniePortals # TODO : this should be split into two cache clearings - clear the standby/incoming cache before the switchover, and clear the outgoing cache after the switchover
+    switchover_ingress_rules_to_updated_database_deployment $MANAGE_DATABASE_TOOL_PROPERTIES_FILEPATH $DESTINATION_COLOR
+    /data/portal-cron/scripts/set_update_process_state.sh /data/portal-cron/pipelines-credentials/manage_genie_database_update_tools.properties complete
```

Comment:
change hard-coded properties filepath to use variable `$MANAGE_DATABASE_TOOL_PROPERTIES_FILEPATH`